### ISSUE TITLE
Incorrect work of addon if there are other files in the directory for rendering

### ISSUE THREAD

**Issue by zhartus** [Fweeb/blender_spritify#7]
The addon can lead Blender to hang if a directory for rendering are already exist some files (images, videos, etc).
Some details can be found here:
http://blenderartists.org/forum/showthread.php?253320-Addon-Spritify&p=3005866&viewfull=1#post3005866

**Comment by DreadKnight** [comment]
Yeah, it's problematic if there are files in the directory, you'll need to manage that yourself carefully for now.
This is kind of a duplicate of my issue here #6
